src/main/java/mate.academy/Main.java

```diff
@@ -10,6 +11,7 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
+        @Inject
         ProductService productService = null;
```

Comment:
The `@Inject` annotation should be used on a field or a constructor, not on a local variable. Move the `@Inject` annotation to a field in the `Main` class and make sure it's placed above the declaration of that field.